server/routes/appointments/video-link-booking/amendRoutes.ts

```diff
@@ -21,6 +23,21 @@ export default function AmendRoutes({ bookAVideoLinkService, prisonService, acti
     return next()
   })
 
```

Comment:
Is there a switch around these A&A routes? Something preventing them turning on before new BVLS is active?